testing/tests/DevExpress.ui.widgets/overlay.tests.js

```diff
@@ -808,6 +808,19 @@ testModule("position", moduleConfig, () => {
         assert.strictEqual($overlayWrapper.css("position"), devices.real().ios ? "absolute" : "fixed");
     });
 
+    test("wrapper should have 100% width and height when shading is disabled", (assert) => {
```

Comment:
Add the `shading` option change test at runtime. And check the `pointer-events` value